roles/kubernetes/node/tasks/pre_upgrade.yml

```diff
@@ -4,3 +4,8 @@
   args:
     creates: "/var/lib/cni"
   failed_when: false
+
+- name: "Pre-upgrade | ensure kubelet container is stopped if using host deployment"
+  command: docker stop kubelet
```

Comment:
@bradbeam, this was only for supporting the forced change from docker->host mode since cri/qos aren't supported for kubelet inside a docker container. Generally, if a user wants to change the deployment type for a service or network plugin, they should do a full reset. The only case we want to interfere is when we modify the defaults